keylime/tenant.py

```diff
@@ -368,6 +392,15 @@ def init_add(self, args):
             raise UserError("Payload size %s exceeds max size %d" % (
                 len(self.payload), config.getint('tenant', 'max_payload_size')))
 
+    def enforce_pcrs(self, policy_pcrs, protected_pcrs, pcr_use) :
+        policy_pcrs = list(self.tpm_policy.keys())
+        policy_pcrs.remove('mask')
+
+        for _pcr in policy_pcrs :
+            if int(_pcr) in protected_pcrs :
+                logger.error(f"WARNING: PCR {_pcr} is specified in \"tpm_policy\", but will in fact be used by {pcr_use}. Please remove it from policy")
```

Comment:
Just to clarify my understanding, does mean users can cherry pick PCRs to either tpm_policy or measure boot (mb)?

Reply:
Correct